runbooks/KubevirtVmHighMemoryUsage.md

```diff
@@ -0,0 +1,26 @@
+# KubevirtVmHighMemoryUsage
+
+## Meaning
+
+Container that hosting Virtual Machine has free memory less than 20 MB and it is close to memory limit.
+
+## Impact
+
+Virtual Machine is at risk of being terminated by the runtime when container memory usage will exceed the memory limit.
```

Comment:
@andreyod 
When the container memory usage will exceed the memory limit, the Virtual Machine is at risk of being terminated by the runtime.

Reply:
Changed